setup.py

```diff
@@ -9,16 +9,22 @@
 from setuptools import find_packages, setup, Command
 
 # Package meta-data.
-NAME = "ross"
```

Comment:
It looks like the current version of the setup.py file (after we used black) has not been merged on this branch and this PR is trying to revert all the changes.
One possibility is to revert the commit related to this modification on the setup.py file. Another possibility would be for you to run black on the setup.py file and commit again.

Reply:
I've just run black on setup.py.
Now it's built just like the setup.py file from master branch.